src/user.ts

```diff
@@ -30,6 +31,32 @@ export class UserModule {
         });
     }
 
+    /**
+     * Find out whether a specified project is available to a currently logged user
+     *
+     * @method isLoggedInProject
+     * @return {Promise} resolves with true if user logged in and project available,
+     *                   resolves with false if user logged in and project not available,
+     *                   rejects if user not logged in
+     */
+    public isLoggedInProject(projectId: string) {
+        return this.getCurrentProfile().then((profile) => {
+            return new Promise((resolve, reject) => {
+                const projectModule = new ProjectModule(this.xhr);
+
+                projectModule.getProjects(profile.links.self.split('/')[4]).then((projects) => {
+                    if (projects.find((p: any) => p.links.self === `/gdc/projects/${projectId}`)) {
+                        resolve(true);
+                    } else {
+                        resolve(false);
+                    }
+                }, (err: ApiResponseError) => {
```

Comment:
I think it would be good to cover rejection as well with a test. But it would be good to mock it with real response as much as possible. what do you think ?